src/components/Layout/_layout.scss

```diff
@@ -1,4 +1,4 @@
-@import "vanilla-framework/scss/settings_colors";
+@import "vanilla-framework/scss/vanilla";
 @import "../../scss/settings";
```

Comment:
Don't you want settings imported before vanilla so it can override variables.

Reply:
They don't currently override any Vanilla settings but I suppose they may in future - I've now swapped them around.